app/views/content/non-uk-teachers/fees-and-funding-for-non-uk-trainees.md

```diff
@@ -77,17 +77,41 @@ Some support may be available if:
 
 ## Teacher training fees
 
-As a non-UK candidate, you can expect to pay between £9,250 and about £36,000 to train to teach in England – fees vary between training providers. You’ll also have to cover your living costs. Most non-UK candidates will not be eligible for financial support to help with fees.
+Teacher training fees for non-UK candidates vary between training providers.
 
-However, if you're an Irish citizen or have settled status, you may get a reduction in your fees to the 'home fee rate' paid by UK applicants (set at a maximum of £9,250). If you're eligible, this fee will be the same for all courses offered by accredited training providers. You may also be eligible for [student loans](#student-loans).  
+Fees:
 
-You can apply for a [salaried teacher training course](/funding-and-support/salaried-teacher-training). On this type of course, you will not pay a fee, and you can earn money while you train in a school. However, these courses are limited in number and very competitive. 
+* cost £36,000 at their most expensive
+* average at £14,765 
```

Comment:
@laurahermionetennant92 one tiny tweak, sorry. In the updated version under Teacher training fees. We don't need to repeat cost in the bullets. It could just say Fees cost:

Fees:

**cost** an average of £14,765
**cost** a maximum of about £36,000